test/kubemark/start-kubemark.sh

```diff
@@ -41,13 +41,14 @@ RESOURCE_DIRECTORY="${KUBEMARK_DIRECTORY}/resources"
 LOCAL_KUBECONFIG="${RESOURCE_DIRECTORY}/kubeconfig.kubemark"
 LOCAL_KUBECONFIG_TMP="${RESOURCE_DIRECTORY}/kubeconfig.kubemark.tmp"
 TP_ONE_KUBECONFIG="${RESOURCE_DIRECTORY}/kubeconfig.kubemark-tp-1"
+TP_TWO_KUBECONFIG="${RESOURCE_DIRECTORY}/kubeconfig.kubemark-tp-2"
 
 ### files to explicitly save the kubeconfig to different cluster or proxy
 ### those are used for targeted operations such as tenant creation etc on
 ### desired cluster directly
 ###
-TP1_KUBECONFIG_SAVED="${RESOURCE_DIRECTORY}/kubeconfig.kubemark.tp1.saved"
-TP2_KUBECONFIG_SAVED="${RESOURCE_DIRECTORY}/kubeconfig.kubemark.tp2.saved"
+TP1_KUBECONFIG_DIRECT="${RESOURCE_DIRECTORY}/kubeconfig.kubemark.tp-1.direct"
+TP2_KUBECONFIG_DIRECT="${RESOURCE_DIRECTORY}/kubeconfig.kubemark.tp-2.direct"
 PROXY_KUBECONFIG_SAVED="${RESOURCE_DIRECTORY}/kubeconfig.kubemark.proxy.saved"
```

Comment:
direct seems to be a better name, can you rename the other two .save file names as well. thanks.